tests/Functional/PlanCommandsTest.php

```diff
@@ -67,29 +67,18 @@ public function testPlanInfoCommand()
      * @covers \Pantheon\Terminus\Commands\Plan\SetCommand
      *
      * @group plan
-     * @group long_fixme
+     * @group long
      */
     public function testSetPanCommand()
```

Comment:
Typo in existing code: "Pan".  Should be `testSetPlanCommand`?  Or just `testSetPlanCommands` to be consistent with `testNewRelicInfoEnableDisableCommands`

Reply:
`testSetPlanCommand` (without "s")

Reply:
It is currently `testSetPanCommand`.  "Pan" - the L is missing.

Reply:
Fixed